yml/3rd_party/acronis/qt5network.yml

```diff
@@ -0,0 +1,17 @@
+---
+Name: qt5network.dll
+Author: Micah Babinski
+Created: 2024-12-17
+Vendor: Acronis
+ExpectedLocations:
+  - '%PROGRAMFILES%\Acronis\CyberProtectHomeOffice'
+VulnerableExecutables:
+  - Path: '%PROGRAMFILES%\common files\acronis\syncagent\syncagentsrv.exe'
+    Type: Sideloading
+    SHA256:
+      - '6f7f390b2012e7dfef9fcbd673a4a0256e2e217b11831e9a27a9d460ba57c0d2'
```

Comment:
Is this SHA correct? It does not seem to belong to an Acronis product

Reply:
Hi Wietze, thanks for the review. Yeah, I was confused by this. The report mentions that Qt5Network.dll was loaded by syncagentsrv.exe, a legitimate application. But it doesn't provide the EXE hash. A quick google search reveals that this EXE name is part of an Acronis product. So I recorded the vendor of the product which the malicious zipfile was likely trying to mimic. When I looked up the [zipfile which contained the EXE and DLL in VT](https://www.virustotal.com/gui/file/7b8958ed2fc491b8e43ffb239cdd757ec3d0db038a6d6291c0fd6eb2d977adc4/relations), I can see that the syncagentsrv.exe was actually the file represented by the hash I provided in the PR: https://www.virustotal.com/gui/file/6f7f390b2012e7dfef9fcbd673a4a0256e2e217b11831e9a27a9d460ba57c0d2.

Based on what little I have found on the actual EXE used in the attack, PasswordChanger.exe (Active@ Password Changer by LSoft Technologies), this would normally be located in:
* `%PROGRAMFILES%\LSoft Technologies\Active@ Password Changer\PasswordChanger.exe`
* `%PROGRAMFILES%\Active Data Recovery Software\Active Password Changer\PasswordChanger.exe` or possibly
* `%PROGRAMFILES%\LSoft Technologies\Active@ Data Studio\PasswordChanger.exe`

Should I update the submission to be geared toward Active@ Password Changer?

Thanks again!